src/main/java/org/apache/mesos/hdfs/state/PersistentState.java

```diff
@@ -22,7 +22,8 @@
   public PersistentState(SchedulerConf conf) {
     MesosNativeLibrary.load(conf.getNativeLibrary());
     this.zkState = new ZooKeeperState(conf.getStateZkServers(),
-        conf.getStateZkTimeout(), TimeUnit.MILLISECONDS, "/hdfs-mesos/" + conf.getClusterName());
+        conf.getStateZkTimeout(), TimeUnit.MILLISECONDS, "/hdfs-mesos/" + conf.getClusterName()
```

Comment:
This is a hack because HA is not working that should not be checked in. This issue will be fixed when the HA changes are done.